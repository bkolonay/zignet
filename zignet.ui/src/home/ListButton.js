import React, { Component } from 'react';

class ListButton extends Component {

  render() {
    return (
      <a href="/list" className="btn btn-primary" role="button" title="View results as list">List</a>
    );
  }
}

export default ListButton;
