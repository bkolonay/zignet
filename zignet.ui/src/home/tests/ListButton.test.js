import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ListButton from '../ListButton';

it('renders', () => {
  const component = <ListButton />;
  ReactDOM.render(component,document.createElement('div'));
});

it('snapshot', () => {
  const component = <ListButton />;
  const tree = renderer.create(component).toJSON();
  expect(tree).toMatchSnapshot();
});