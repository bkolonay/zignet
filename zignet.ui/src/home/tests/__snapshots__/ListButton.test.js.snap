// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<a
  className="btn btn-primary"
  href="/list"
  role="button"
  title="View results as list"
>
  List
</a>
`;
